config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: registry.hub.docker.com/secrets-manager:v1.0.0-snapshot-1
+      - image: docker.tuenti.io/sre/secrets-manager:v1.0.0-rc.1
```

Comment:
this is a private registry, not sure that should be here.

Reply:
right this is due to some tests I did and I forget to remove it. Thx.